crates/flox/src/commands/channel.rs

```diff
@@ -28,6 +30,8 @@ use crate::subcommand_metric;
 use crate::utils::dialog::{Dialog, Select, Text};
 use crate::utils::init::{DEFAULT_CHANNELS, HIDDEN_CHANNELS};
 
+const SEPARATOR: &'_ str = ":";
```

Comment:
```suggestion
const SEARCH_TERM_SEPARATOR: &'_ str = ":";
```

separator is quite generic, lets be a bit more precise

Reply:
Fixed in #284